js/testapps/firebase-functions-sample1/functions/src/index.ts

```diff
@@ -14,66 +14,48 @@
  * limitations under the License.
  */
 
-import { firebase } from '@genkit-ai/firebase';
-import { firebaseAuth } from '@genkit-ai/firebase/auth';
-import { noAuth, onFlow } from '@genkit-ai/firebase/functions';
+import { enableFirebaseTelemetry } from '@genkit-ai/firebase';
 import {
   FirebaseUserEngagementSchema,
   collectUserEngagement,
 } from '@genkit-ai/firebase/user_engagement';
-import { geminiPro, vertexAI } from '@genkit-ai/vertexai';
-import { onRequest } from 'firebase-functions/v2/https';
-import { genkit, run, z } from 'genkit';
+import { gemini10Pro, vertexAI } from '@genkit-ai/vertexai';
+import { onCallGenkit, onRequest } from 'firebase-functions/v2/https';
```

Comment:
what's the preferred import, `firebase-functions/https` or `firebase-functions/v2/https`?